### PR TITLE
feat: Drift: More resources

### PR DESCRIPTION
**Pull request by disq**
also see https://github.com/cloudquery/cloudquery/pull/563

### CODE REVIEW

File: resources/provider/moduledata/drift/2/drift.hcl [hunk 1]
```diff
@@ -0,0 +1,1913 @@
...
+    filters = [ "c.is_default!=true" ]
+
+    iac {
+      terraform {
+        type = "aws_network_acl"
+      }
+    }
+  }
+
+  #        resource "aws_ec2_network_acl_entries" {
```
Comment (roneli) [inline resources/provider/moduledata/drift/2/drift.hcl:886]: why is this commented out?

File: resources/provider/moduledata/drift/2/drift.hcl [hunk 2]
```diff
@@ -0,0 +1,1913 @@
...
+    ignore_attributes = [ "ebs_encryption_enabled_by_default" ]
+    iac {
+      terraform {
+        type = "aws_ebs_default_kms_key"
+        identifiers = [ "key_arn" ]
+      }
+    }
+  }
+
+  resource "ec2.regional_config#ebs" { # TODO: add account/region support
```
Comment (roneli) [inline resources/provider/moduledata/drift/2/drift.hcl:909]: open issue for these ?

**Review by roneli — APPROVED** [review]
LGTM, can we create an issue with a Task list based form the todo here etc'?

File: resources/provider/moduledata/drift/2/drift.hcl (hunk 1 above)
Comment (disq) [reply resources/provider/moduledata/drift/2/drift.hcl:886]: Look one line down, it says:
`TODO: no CRC32 function, no data in tests to verify`

File: resources/provider/moduledata/drift/2/drift.hcl (hunk 2 above)
Comment (disq) [reply resources/provider/moduledata/drift/2/drift.hcl:909]: https://github.com/cloudquery/cloudquery/issues/566

**Comment by disq** [comment]
Implements cloudquery/cloudquery#1406

**Pull request merged**